src/Tribe/Aggregator/Cron.php

```diff
@@ -87,9 +87,9 @@ public function get_frequency( $search = array() ) {
 		 */
 		$found = $schedules = apply_filters( 'tribe_aggregator_record_frequency', array(
 			(object) array(
-				'id'       => 'inactive',
+				'id'       => 'on_demand',
 				'interval' => false,
-				'text'     => esc_html_x( 'Inactive (manual/on demand)', 'aggregator schedule frequency', 'the-events-calendar' ),
+				'text'     => esc_html_x( 'On demand', 'aggregator schedule frequency', 'the-events-calendar' ),
```

Comment:
I think this should be `On Demand` rather than the sentence case of `On demand` as per the Central ticket.